svo (setup, misc, empty, funnies, dor).xml

```diff
@@ -512,7 +512,7 @@ signals.gmcpcharafflictionsremove:connect(function()
   if thisaff:sub(-4) == " (1)" then thisaff = thisaff:sub(1, -5) end
   gaffl[thisaff] = nil
   if conf.gmcpaffechoes then svo.echof("Cured aff %s", thisaff) end
-  if svo.dict.unknownany.count >= 1 and not svo.affl[thisaff] then
+  if svo.dict.unknownany.count &gt;= 1 and not svo.affl[thisaff] then
```

Comment:
I didn't notice when this got changed - this came out of an older file that I've not updated in a while... Apparently someone messed up this particular spot in the code. &gt; is the correct way for a > to show up in an xml!!